modules/cliparser.sh

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright 2020 Northern.tech AS
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+function parse_cli_options () {
+  # The mender_convert_config is always used and provides all the defaults
```

Comment:
This comment looks out of place now. Maybe it was earlier as well :)

Reply:
Fixing :)